docker/docker-entrypoint.sh

```diff
@@ -19,8 +19,12 @@ mysql_error() {
 	mysql_log ERROR "$@" >&2
 	exit 1
 }
+docker_process_sql() {
+  dolt sql
```

Comment:
I think it's ok to just start dolt as a commandline for sake initialization. Pros of this approach is that the container isn't listen to TCP during the initialization, that is good for e. g. readiness probes